cmd/health/main.go

```diff
@@ -63,7 +63,7 @@ func main() {
 		log.SetLevel(log.ErrorLevel)
 	}
 
-	gateway, err := kiamserver.NewGateway(opts.serverAddress, opts.serverAddressRefresh, opts.caPath, opts.certificatePath, opts.keyPath)
+	gateway, err := kiamserver.NewGateway(opts.serverAddress, opts.serverAddressRefresh, 50*time.Millisecond, opts.caPath, opts.certificatePath, opts.keyPath)
```

Comment:
As with the agent's main, we should construct a context and pass it to `NewGateway` and set the timeout on that context from a cli flag.